databricks-cluster-log-permissions/providers.tf

```diff
@@ -0,0 +1,10 @@
+provider "aws" {
```

Comment:
I *think* this will work

Original: https://github.com/chanzuckerberg/shared-infra/blob/95541fe5d66de9f180303236fed141d65d2ce33c/terraform/modules/databricks-cluster-log-permissions/providers.tf